app/models/service_draft.rb

```diff
@@ -7,7 +7,7 @@ def forms
       []
     end
 
-    def create_from_service(service, user, native_id)
+    def create_from_service(service, user, native_id, concept_id=nil)
```

Comment:
This is minor, but since keyword arguments are used in the method call for `find_or_initialize_by`, you might consider using them for this `create_from_service` method

Reply:
I appreciate the suggestion. I do think we should make some type of plan to convert most of our methods to use keyword arguments more systematically, but for this ticket I tried to not make that conversion unless it was necessary.